content/community/certification.md

```diff
@@ -11,15 +11,20 @@ Reviewer: Tim Sutton
 
 {{< content-start >}}
 
-# Become a QGIS certified training organisation
-Is your organisation an active member of the QGIS community and offering high-quality training courses with QGIS? Then consider applying for the **QGIS Certification Programme**. As a member, you'll be able to issue the official QGIS certificate to participants of your courses, workshops and training. Each certificate contributes €20 to QGIS.
+# Certification Programme
+
+## QGIS Certificate
+
 
-If you're looking for organisations that offer training with the official QGIS certificate, check the [current list of members](https://changelog.qgis.org/en/qgis/certifyingorganisation/list/) of the QGIS Certification Programme.
+If you're looking for organisations that offer training with the official QGIS certificate, check the [current list of members](https://certification.qgis.org/en/certifyingorganisation/list/) of the QGIS Certification Programme.
 
 ![Students with certificate](../students_with_certificate.webp "Students with certificate")
 
-## Certification Process
-### Requirements
+## Become a QGIS certified training organisation
+Is your organisation an active member of the QGIS community and offering high-quality training courses with QGIS? Then consider applying for the **QGIS Certification Programme**. As a member, you'll be able to issue the official QGIS certificate to participants of your courses, workshops and training. Each certificate contributes €20 to QGIS.
```

Comment:
Does your organisation participate actively in the QGIS community and offer high-quality training courses with QGIS? Then consider applying for the **QGIS Certification Programme**. As a member, you'll be able to issue the official QGIS certificate to participants of your courses, workshops and training. Each certificate contributes €20 to QGIS.